sites/public/src/lib/hooks.ts

```diff
@@ -103,36 +91,7 @@ export async function fetchBaseListingData({
       params.orderDir = orderDir
     }
 
-    if (process.env.bloomJurisdictionNames.length != 0) {
-      // This function early returns if the jurisdictions have already been.
-      // fetched from the Bloom API.
-      const jurisdictions = await fetchBloomJurisdictionsByName()
-      params.bloomJurisdiction = jurisdictions.map((jurisdiction) => jurisdiction.id)
-
-      const response = await axios.get(process.env.listingsWithExternalServiceUrl, {
-        params,
-        paramsSerializer: (params) => {
-          return qs.stringify(params)
-        },
-      })
-      const listingsWithExternal = response.data
-      let allListings: ListingWithSourceMetadata[] = listingsWithExternal.local.items.map(
-        (item: ListingWithSourceMetadata) => {
-          item.isBloomListing = false
-          return item
-        }
-      )
-      for (const k in listingsWithExternal.external) {
-        allListings = allListings.concat(
-          listingsWithExternal.external[k].items.map((item: ListingWithSourceMetadata) => {
-            item.isBloomListing = true
-            return item
-          })
-        )
-      }
-      return allListings
-    }
-    const response = await axios.get(process.env.listingServiceUrl, {
+    const response = await axios.get(getListingServiceUrl(), {
```

Comment:
IMO I think its safer if we maintain the separation between the including other external table listings url and the legacy /listings endpoint (that is, define new env variables for the new endpoint and explicitly replace this with that)

we will inadvertently break fewer things, and there are use cases on the public site for the non-external  listings only (e.g. applications)

Reply:
Other than this function, which specifically queries a list of Listings, the only place where `process.env.listingServiceUrl` was used is in a large commented out section of code in `[slug].tsx`.  I'm not opposed to having separate functions/vars for "query endpoint" vs "CRUD endpoint", but nothing exists yet that would call that function or use that var.  Would it be better if I renamed it to something like `getListingServiceQueryEndpoint()`?  I'm planning on consolidating runtime configuration in a later PR, FWIW

Reply:
I see, if this is the only place this is used thats fine, we may also want to delete the /listings endpoint altogether at that point (or just update /listings directly)

Reply:
It looks like `[slug].tsx` builds the "findOne" URL directly from a hardcoded value of `/listings/` on line 125.  That's not great, but we can look at addressing that in a follow-up PR